api/specs.proto

```diff
@@ -84,7 +84,7 @@ message ServiceSpec {
 		// Aliases specifies a list of discoverable alternate names for the service on this Target.
 		repeated string aliases = 2;
 	}
-	repeated NetworkAttachmentConfig networks = 7;
+	repeated NetworkAttachmentConfig networks = 7 [deprecated=true];
```

Comment:
> Should it be migrated once and for all or should we have code able to deal with both?

We should handle and broadcast both.

Really, the deprecation plan should be placed on the field description.